sdk/aqueduct/integrations/utils.py

```diff
@@ -0,0 +1,100 @@
+from typing import Any, Dict, List, Mapping, Optional, Union
+
+from aqueduct.constants.enums import OperatorType
+from aqueduct.error import AqueductError
+from aqueduct.models.dag import DAG
+from aqueduct.models.operators import Operator
+
+
+def _generate_extract_op_name(
+    dag: DAG,
+    integration_name: str,
+    name: Optional[str],
+) -> str:
+    """
+    Generates name for extract operators to avoid operators with the same name.
+
+    Arguments:
+        dag:
+            DAG that operator will be a part of.
+        integration_name:
+            Name of integration to run extract on.
+        name:
+            Optinally provided operator name.
+    Returns:
+        Name for extract operator.
+    """
+
+    op_name = name
+
+    default_op_prefix = "%s query" % integration_name
+    default_op_index = 1
+    while op_name is None:
+        candidate_op_name = default_op_prefix + " %d" % default_op_index
+        colliding_op = dag.get_operator(with_name=candidate_op_name)
+        if colliding_op is None:
+            op_name = candidate_op_name  # break out of the loop!
+        default_op_index += 1
+
+    assert op_name is not None
+
+    return op_name
+
+
+def _get_checks_for_op(op: Operator, dag: DAG) -> List[Operator]:
```

Comment:
Do these utils functions belong in this package? I think they would be better suited in the `aqueduct.artifacts` package.